client/src/components/formHelpers/form.tsx

```diff
@@ -1,24 +1,75 @@
 import React, { FormEvent } from 'react';
 import { Form } from 'react-final-form';
+import normalizeUrl from 'normalize-url';
 
 import {
-  URLValues,
-  ValidatedValues,
-  FormFields,
-  BlockSaveButton,
-  BlockSaveWrapper,
-  formatUrlValues
-} from '../formHelpers/index';
+  localhostValidator,
+  editorValidator,
+  composeValidators,
+  fCCValidator,
+  httpValidator
+} from './form-validators';
+import FormFields, { FormOptions } from './form-fields';
 
-export type FormOptions = {
-  ignored?: string[];
-  isEditorLinkAllowed?: boolean;
-  isLocalLinkAllowed?: boolean;
-  required?: string[];
-  types?: { [key: string]: string };
-  placeholders?: { [key: string]: string };
+import { default as BlockSaveButton } from './block-save-button';
+import { default as BlockSaveWrapper } from './block-save-wrapper';
+
+type URLValues = {
+  [key: string]: string;
+};
+
+type ValidationError = {
+  error: { message?: string };
+  value: string;
+};
+
+export type ValidatedValues = {
+  values: URLValues;
+  errors: ValidationError[];
+  invalidValues: (JSX.Element | null)[];
 };
 
+const normalizeOptions = {
+  stripWWW: false
+};
+
+function formatUrlValues(
```

Comment:
Couple of things stand out.  Can this be moved into a separate file?  Maybe just called `form-helpers`.  It seems a little out of place here.

Also, how about `dynamic-form` for the directory name, rather than `formHelpers`?  The latter makes it sound like things you would need when buildling a form, rather than the form itself (which is really what's being exported here)